src/sentry/static/sentry/app/components/group/externalIssueForm.tsx

```diff
@@ -60,9 +67,25 @@ class ExternalIssueForm extends AsyncComponent<Props, State> {
     ];
   }
 
+  startTransaction = (type: 'load' | 'submit') => {
+    const {action, group, integration} = this.props;
+    const transaction = Sentry.startTransaction({name: `externalIssueForm.${type}`});
```

Comment:
Would you be able to get the information you want by having a single transaction and making the type/integration slug tags on the transaction?

Reply:
@markstory  A single transaction for load and submit together?